requirements-dev.txt

```diff
@@ -10,6 +10,15 @@ coverage
 codecov
 # used for example confluence attach file
 python-magic
+pylint
+mypy>=0.812
+bandit
+doc8
+types-Deprecated
+types-requests
+types-six
+types-beautifulsoup4
```

Comment:
All of the above get used in tox so it only made sense to include them in the dev reqs.